### PR TITLE
Enable descriptions in planned quests

### PR DESCRIPTION
**Pull request by eob**

### CODE REVIEW

File: src/agents/quest_agent.py [hunk 1]
```diff
@@ -101,9 +101,16 @@ def run(self, context: AgentContext) -> Action:  # noqa: C901
...
+                if (
+                    quest_description.description
+                    and quest_description.description.strip()
+                ):
+                    optional_desc = f"\n\nAuthor's notes for this quest are: {quest_description.description}"
+
                 context.chat_history.append_system_message(
                     text=f"{game_state.player.name} is embarking on a quest to {quest_description.goal} "
-                    f"at {quest_description.location}.",
+                    f"at {quest_description.location}.{optional_desc}",
```
Comment (dkolas) [inline src/agents/quest_agent.py:113]: suggestion: add a space or newline

**Review by dkolas — APPROVED** [review]
One small suggestion :)

File: src/agents/quest_agent.py (hunk 1 above)
Comment (eob) [reply src/agents/quest_agent.py:113]: The newline is added within the `optional_desc` string so that without it we won't have trailing whitespace

**Review by douglas-reid — COMMENTED** [review]
LGTM

**Pull request merged**